test/distributed/pipelining/test_backward.py

```diff
@@ -19,7 +23,9 @@
 
 
 class StageBackwardTests(TestCase):
-    def test_stage_backward(self, device):
+    @precisionOverride({torch.float: 1e-4})
```

Comment:
We will need to upstream this branch in next step, so the test case should work for all the backends. We should not change the test scope for cuda and other backends.